libs/utils/utils.js

```diff
@@ -659,8 +659,7 @@ export function decorateLinks(el) {
   decorateImageLinks(el);
   const anchors = el.getElementsByTagName('a');
   const { hostname } = window.location;
-  convertStageLinks({ anchors, config, hostname });
-  return [...anchors].reduce((rdx, a) => {
+  const result = [...anchors].reduce((rdx, a) => {
```

Comment:
The result is a list of links, so why not
```suggestion
  const links = [...anchors].reduce((rdx, a) => {
```